internal/repositories/equipment.go

```diff
@@ -637,17 +632,6 @@ func (r *equipmentRepository) UnblockEquipment(ctx context.Context, id int) erro
 	return err
 }
 
-func checkDates(start *time.Time, end *time.Time) (*time.Time, *time.Time, error) {
```

Comment:
Why did you remove this function?

Reply:
THis validation was moved into handler to respond with proper error code and message:
```
		startDate := time.Time(s.Data.StartDate)
		endDate := time.Time(s.Data.EndDate)
		if startDate.After(endDate) {
			return equipment.NewBlockEquipmentDefault(http.StatusBadRequest).
				WithPayload(buildBadRequestErrorPayload(messages.ErrStartDateAfterEnd, ""))
		}
```